ethereum_history_api/tests/src/get_header.e2e.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, expect, it } from 'vitest';
+import { copy, updateNestedField, incHexStr } from 'noir-ethereum-api-oracles';
+import { abiEncode, WitnessMap } from '@noir-lang/noirc_abi';
+
+import { privateKeyToAccount } from 'viem/accounts';
+import getHeaderVerifier from '../../contracts/out/GetHeaderUltraPLONKVerifier.sol/UltraVerifier.json';
+import { Hash } from 'viem';
+import { createAnvilClient } from './ethereum/anvilClient.js';
+import { get_header_circuit, readInputMap, readProof, readWitnessMap } from './main.js';
+import {
+  ANVIL_TEST_ACCOUNT_PRIVATE_KEY,
+  FoundryArtefact,
+  VERIFICATION_GAS_LIMIT,
+  deployVerificationContract,
+  verifyStorageProofInSolidity
+} from './solidityVerifier.js';
+
+describe('get_header', async () => {
+  const account = privateKeyToAccount(ANVIL_TEST_ACCOUNT_PRIVATE_KEY);
+  const client = createAnvilClient();
+  const PROOF_PATH = '../../proofs/get_header.proof';
+  const INPUT_MAP_PATH = '../circuits/get_header/Verifier.toml';
```

Comment:
There is inconsistency where test data is coming from for success and fail.
Given that we only get "Execution reverted" error in the end it is hard to reason if the test are failing for the right reason. 
To add insult to injury file `../../proofs/get_header.proof` is non existing in repository, so if file format changes, there is no easy way to catch it.
Can we make it more consistent?

Reply:
Also in `get_account.e2e`)